packages/swap/src/event-handlers/swapExecuted.ts

```diff
@@ -15,16 +16,34 @@ export default async function swapExecuted({
   block,
   event,
 }: EventHandlerArgs): Promise<void> {
-  const { swapId, egressAmount, intermediateAmount } = swapExecutedArgs.parse(
+  const { swapId, intermediateAmount, egressAmount } = swapExecutedArgs.parse(
     event.args,
   );
+  const swap = await prisma.swap.findUnique({
+    where: { nativeId: swapId },
+  });
+
+  // skip update if we are not tracking swap
+  if (!swap) {
+    return;
+  }
```

Comment:
we should be tracking all swaps now

Reply:
i think so, right? i was scared about breaking something but will remove the check